jolt-core/src/poly/commitment/commitment_scheme.rs

```diff
@@ -136,37 +135,26 @@ pub trait CommitmentScheme: Clone + Sync + Send + 'static {
 }
 
 pub trait StreamingCommitmentScheme: CommitmentScheme {
+    /// The type representing chunk state (tier 1 commitments)
     type ChunkState: Send + Sync + Clone + PartialEq + Debug;
-    type CachedData: Sync;
 
-    /// Prepare cached data that will be shared across all polynomial commitments
-    fn prepare_cached_data(setup: &Self::ProverSetup) -> Self::CachedData;
-
-    // First `process_chunk` is a tier 1 commitment (MSM)
-
-    /// Process a chunk of small scalar values
-    fn process_chunk<T: SmallScalar>(
-        cached_data: &Self::CachedData,
+    /// Compute tier 1 commitment for a chunk of small scalar values
+    fn compute_tier1_commitment<T: crate::utils::small_scalar::SmallScalar>(
+        setup: &Self::ProverSetup,
         chunk: &[T],
     ) -> Self::ChunkState;
 
-    /// Process a chunk of field elements
-    fn process_chunk_field(
-        cached_data: &Self::CachedData,
-        chunk: &[Self::Field],
-    ) -> Self::ChunkState;
-
-    /// Process a chunk of one-hot values
-    fn process_chunk_onehot(
-        cached_data: &Self::CachedData,
+    /// Compute tier 1 commitment for a chunk of one-hot values
+    fn compute_tier1_commitment_onehot(
```

Comment:
I kinda liked `process_chunk` and `finalize` better than `compute_tier1_commitment` and `compute_tier2_commitment`; it seems less Dory-specific